ls1intum/Artemis
# `Quiz exercises`: Disable preview button for DnD questions in case no background image is uploaded

### Checklist
#### General


#### Client

### Motivation and Context


Fixes #5156. Fixes #1392.

### Description

To fix #5156, the preview button is disabled until a background image is uploaded. Additionally, a tooltip on the disabled button informs the user that they have to upload a background image first in order to show a preview.

### Steps for Testing

Prerequisites:
- 1 Instructor

1. Create a new quiz exercise
2. Add a new Drag and Drop question
3. Without uploading any background image, the preview button should be disabled and a tooltip should be displayed.
4. Upload a background image5. 
5. The preview button should now be enabled
6. Click the preview button and make sure the actual preview is displayed

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage


| Class/File | Branch | Line |
|------------|-------:|-----:|


### Screenshots



Screenshot showing the disabled button and the corresponding tooltip
<img width="1545" alt="Bildschirmfoto 2022-05-31 um 16 09 42 (2)" src="https://user-images.githubusercontent.com/9954674/171194057-968eabe5-7c0a-46ea-b424-5f275eee490e.png">
